### ISSUE TITLE
Toolbar Example: Add missing descriptive tooltips

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#986]
Per [comment in issue 539](https://github.com/w3c/aria-practices/issues/539#issuecomment-454870090)
by @sh0ji about the [toolbar example](http://w3c.github.io/aria-practices/examples/toolbar/toolbar.html),
all the controls should have a descriptive title attribute so sighted users don't have to infer or discover their function.
Currently, some have tooltips and some do not.

**Comment by lenoraporter** [comment]
Anyone working on this task?

**Comment by ardnahcivar** [comment]
@lenoraporter , yes .

**Comment by carmacleod** [comment]
@jongund Here's what I see on focus (popup label's "tail" is perfectly joined to the text bubble):
![image](https://user-images.githubusercontent.com/3331913/59712336-0177bb00-91db-11e9-9e11-6447341f89d4.png)

And here's what I see on hover (popup label's "tail" is slightly separated from the text bubble):
![image](https://user-images.githubusercontent.com/3331913/59712231-d1301c80-91da-11e9-9e11-b4b98b3c4691.png)

I'm on Windows 10, latest Chrome. I'll see if I can debug it.

**Comment by carmacleod** [comment]
@jongund 
I can make the above problem go away by making any one of the following 3 CSS changes.
Your choice which one you like best.

**1.** change the 2px white border to be 1px in:
```css
[role="toolbar"] button .popup-label {
    ...
    border: 1px solid white;
    ...
}
```

**OR
2.** change the top/bottom padding from 2px to 1px in:
```css
[role="toolbar"] button .popup-label {
    ...
    padding: 1px 4px;
    ...
}
```

**OR** (this one is a bit strange... ;)
**3.** change the top from 100% to 99% in:
```css
[role="toolbar"] button .popup-label:after, [role="toolbar"] button .popup-label:before {
    top: 99%;
    ...
}
```

**Comment by jongund** [comment]
@carmacleod Carolyn thanks for the fixes I like the first option the best, I will make the change.

**Comment by jongund** [comment]
@carmacleod Do you want to check the updated example:
(Toolbar Example from GitHub branch)[https://raw.githack.com/w3c/aria-practices/issue986-toolbar-button-tooltips/examples/toolbar/toolbar.html]

**Comment by carmacleod** [comment]
@jongund Works perfectly! Thank-you!

**Comment by mcking65** [comment]
Resolved with commit 19f3cdf from merging pr #1069. Thank you @jongund!! Thank you also to the several other people who contributed ideas and feedback!
